### PR TITLE
Auto register API Key once it's retrieved from Subnet and minor refactor

### PR DESCRIPTION
**Pull request by reivaj05**

### CODE REVIEW

File: portal-ui/src/screens/Console/Support/Register.tsx [hunk 1]
```diff
@@ -595,15 +607,10 @@ const Register = ({ classes }: IRegister) => {
...
           />
         ) : (
-          <ApiKeyRegister
-            classes={classes}
-            setApiKey={setLicense}
-            apiKey={license}
-            loading={loading}
-            onRegister={subnetLogin}
-          />
+          <ApiKeyRegister classes={classes} afterRegister={fetchLicenseInfo} />
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Support/Register.tsx:610]: Classes shouldn't be passed this way. Please implement HOC in ApiKeyRegister or load them using `useStyles` hook.

**Review by bexsoft — CHANGES_REQUESTED** [review]

**Comment by cniackz** [comment]
All good, except we can't decrease our coverage:

Screen Shot 2022-07-27 at 7 29 16 PM

**Review by cniackz — COMMENTED** [review]
All good, except we can't decrease our coverage.

**Comment by cniackz** [comment]
As this is FE change only, please adjust Threshold accordingly.
Not sure why this is affected. But we can figure that later on.

**Review by bexsoft — DISMISSED** [review]
Could you please squash all the commits in a single one?  Thank you!

**Comment by reivaj05** [comment]
new PR https://github.com/minio/console/pull/2217

**Pull request closed without merging**